.gitignore

```diff
@@ -2,5 +2,7 @@ sync/config
 sync/windows/bin/
 sync/linux/bin/
 sync/shared/config
+sync/shared/kubejoin.ps1
```

Comment:
These are automatically generated scripts, so ignoring it.

Reply:
FYI, I kept  `sync/shared/kubejoin.ps1` so far from `.gitignore` because every file that is referenced in the Vagrantfile must at least _exist,_ otherwise vagrant will complain.